pkg/operator/controller/names.go

```diff
@@ -147,6 +147,10 @@ func LoadBalancerServiceName(ic *operatorv1.IngressController) types.NamespacedN
 	return types.NamespacedName{Namespace: "openshift-ingress", Name: "router-" + ic.Name}
 }
 
+func NodePortServiceName(ic *operatorv1.IngressController) types.NamespacedName {
+	return types.NamespacedName{Namespace: "openshift-ingress", Name: "router-nodeport-" + ic.Name}
```

Comment:
There's an opportunity here to start undoing the "router" naming convention — or we could remain consistent. Just wanted to point it out.